Error handling copy improvements (#115)
rtk-incubator/rtk-query: docs/concepts/mutations.md

      <EditablePostName
        name={post.name}
        onUpdate={(name) => {

>           // Being that RTK Query uses createAsyncThunk from RTK under the hood,

Reviewer: ```suggestion
          // Since RTK Query uses createAsyncThunk from RTK under the hood,
```
